docker/pykoi-cpu-custom/app.py

```diff
@@ -0,0 +1,34 @@
+"""Demo for the chatbot application using OpenAI endpoint."""
+import pykoi
+
+##########################################################
+# Creating an OpenAI model (requires an OpenAI API key) #
+##########################################################
+# enter openai api key here
+api_key = "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"
+
+# Creating an OpenAI model
+model = pykoi.ModelFactory.create_model(
+    model_source="openai",
+    api_key=api_key)
+
+#####################################
+# Creating a chatbot with the model #
+#####################################
+database = pykoi.QuestionAnswerDatabase(debug=True)
+chatbot = pykoi.Chatbot(model=model, feedback="vote")
+# chatbot = pykoi.Chatbot(model=model, feedback="rank")
+dashboard = pykoi.Dashboard(database=database)
+
+###########################################################
+# Starting the application and add chatbot as a component #
+###########################################################
+# Create the application
+# app = pykoi.Application(debug=False, share=True)
+app = pykoi.Application(
+    host='0.0.0.0',
```

Comment:
can you give for example "127.0.0.1" or the default value a try? Maybe this port was in use to cause the issue regarding not accessible.

Reply:
Have reverted to default setup